README.md

```diff
@@ -67,6 +67,14 @@ or via Kubeconfig:
 helmClient, err := helmclient.NewClientFromKubeConf()
 ```
 
+## Mock Client
+
+This library includes a mock client [mock/interface_mock.go](./mock/interface_mock.go) which is generated by [mockgen](https://github.com/golang/mock).
+
+Example usage of the mocked client can be found in [mock/mock_test.go](./mock/mock_test.go).
+
+If you made changes to [interface.go](./interface.go), you should issue the `go generate ./...` command to trigger code generation. 
```

Comment:
Instead of "just" mentioning this, maybe we should verify this in the CI workflow (have it fail when someone forgot to re-generate the code)?

```bash
go generate ./... && test $(git status --porcelain | wc -l) -eq 0
```